core/src/main/java/com/sequenceiq/cloudbreak/controller/ImageCatalogController.java

```diff
@@ -19,9 +29,72 @@
     @Inject
     private ImagesToImagesResponseJsonConverter converter;
 
+    @Inject
+    private AuthenticatedUserService authenticatedUserService;
+
+    @Inject
+    private ImageCatalogToResponseConverter toResponseConverter;
```

Comment:
We could use one conversion service instance instead of many exact converter instance.

`
@Inject
@Named("conversionService")
private ConversionService conversionService;
`